### ISSUE TITLE
display ANI in search results?

### ISSUE THREAD

**Issue by ctb** [sourmash-bio/sourmash#2001]
Once https://github.com/sourmash-bio/sourmash/pull/1967 is merged, ANI will available in CSV files! 🎉 

It is also available in `sourmash compare` matrix output, if `--ani` is used. 🎉 

I don't _think_ it is displayed anywhere else.

Do we want to add ANI to the `search` output, @bluegenes? I'm in favor - the search results are pretty sparse so I think we even have room for them.

Not sure about gather, though. I think the k-mer overlap approaches makes more sense, maybe? But it would be nice to have as an option, maybe? 🤔

Maybe do it for search first, since I'm pretty sure that's a good idea, and then a separate PR (no hurry) for gather?

**Comment by bluegenes** [comment]
ANI to `search` output would be great, but a few thoughts:
- we can't estimate `ANI` for `num` signatures, so `num` vs `scaled` outputs would be different (maybe they already are?)
- `search` uses `Jaccard` by default, which isn't as good as containment for ANI. Does adding ANI to the search output encourage use of jaccard--> ANI?
- How do we want to handle excessive jaccard error? Currently emit a warning, do we want to zero out?
- Do we want to allow ANI estimation for `abund` searches (ANI ignores abundance)? If so, from `jaccard`?

**Comment by ctb** [comment]
> we can't estimate `ANI` for `num` signatures, so `num` vs `scaled` outputs would be different (maybe they already are?)

ahh, I'd kinda forgotten that.

**Comment by bluegenes** [comment]
> ahh, I'd kinda forgotten that.

we could actually estimate ANI from `num` sketches using the Mash Distance, but I'd rather not, because: 
1. don't want to confuse folks / have them use the ANI's interchangeably
2. want to encourage transition to FracMinHash

**Comment by ctb** [comment]
agreed.
